packages/mds-policy-service/service/validators/intents.ts

```diff
@@ -40,7 +47,36 @@ validatePermittedParkingAjv.addKeyword({
     if (rule_fields.length < 2) return false
     if (rule_fields[rule_fields.length - 1].maximum !== 0) return false
     const firstNMinusOneRules = rule_fields.slice(0, -1)
-    return firstNMinusOneRules.every((rule: PermittedParkingIntentRuleUserFieldsBase) => rule.geographies.length === 1)
+    return firstNMinusOneRules.every((rule: PermittedParkingIntentRuleUserFields) => rule.geographies.length === 1)
+  }
+})
+
+const checkFinalProviderRebalancingRuleAjv = new Ajv({ allErrors: true })
+checkFinalProviderRebalancingRuleAjv.addKeyword({
+  keyword: 'validateFinalProviderRebalancingRule',
+  type: 'object',
+  schemaType: 'boolean',
+  compile: () => data => {
+    const { rule_fields } = data
+    if (rule_fields.length < 2) return false
+    const firstNMinusOneRules = rule_fields.slice(0, -1)
+    if (
+      !firstNMinusOneRules.every(
+        (rule: ProviderRebalancingZoneIntentRuleUserFields) =>
+          rule.states === null || Object.keys(rule.states).length === 0
+      )
+    )
+      return false
+
+    if (!firstNMinusOneRules.every((rule: PermittedParkingIntentRuleUserFields) => rule.geographies.length === 1))
+      return false
+
+    const finalRule = rule_fields[rule_fields.length - 1]
+    if (finalRule.maximum !== 0) return false
+
+    return MICRO_MOBILITY_VEHICLE_STATES_v1_1_0.every(
```

Comment:
Do we wanna use the `MICRO_MOBILITY_VEHICLE_STATES_v1_1_0` list directly here, or should we use the aliased `MICRO_MOBILITY_VEHICLE_STATES` array so it is automatically updated once that alias goes to a newer version?